pkg/storage/job.go

```diff
@@ -48,19 +52,22 @@ func (jsm JobStorageManager) StoreJobReport(report *job.JobReport) error {
 	return nil
 }
 
-// GetJobReport fetches a job report to the storage layer
+// GetJobReport fetches a job report from the storage layer
 func (jsm JobStorageManager) GetJobReport(jobID types.JobID) (*job.JobReport, error) {
-	return storage.GetJobReport(jobID)
+	report, err := storage.GetJobReport(jobID)
+	if err != nil {
+		return nil, err
+	}
+	return report, nil
```

Comment:
isn't this exactly the same as:
```suggestion
	return storage.GetJobReport(jobID)
```